bitrise.yml

```diff
@@ -196,28 +268,17 @@ workflows:
             exec unzip "${BITRISE_ARTIFACT_PATHS}"
             echo "show dir"
             ls
-    - script@1.1:
-        run_if: '{{getenv "RUN_UI_TESTS" | eq "Run_UI_Tests"}}'
-        title: Test without Building
-        inputs:
-        - content: |
-            #!/usr/bin/env bash
-            set -ex
-
-            ls ./Users/vagrant/git/Tests/
-            cp -r ./Users/vagrant/git/Tests/* .
-            mv ./Users/vagrant/git/DerivedData .
-            ls -la
-
-            echo "-- test-without-building --"
-            xcodebuild -resultBundlePath "xcodebuild.xcresult" -derivedDataPath "/Users/vagrant/git/DerivedData" -destination "platform=iOS Simulator,name=iPhone 14,OS=latest" test-without-building -xctestrun "/Users/vagrant/git/DerivedData/Build/Products/Fennec_Enterprise_XCUITests_SmokeXCUITests_iphonesimulator16.1-x86_64.xctestrun"
-    - custom-test-results-export@0:
-        run_if: '{{getenv "RUN_UI_TESTS" | eq "Run_UI_Tests"}}'
+    - xcode-test-without-building@0:
```

Comment:
Does this need to be renamed, since there's a prebuilt BR step like mentioned earlier?

Reply:
Not sure I understand the question.. this is the name of the pre-built step when it is added to the workflow.
![Captura de pantalla 2023-01-30 a las 20 50 36](https://user-images.githubusercontent.com/1897507/215580756-71bdc370-efea-4d35-b454-5bedecd00845.png)

Reply:
it was a script step before, but now it can be selected. It is a bit clearer and usually more intuitive than  a script step

Reply:
Oh! okay I understand now - this is fine then!